FAQ.md

```diff
@@ -211,6 +211,62 @@ should have an underscore `_` prepended to it.
 (For the last one, that makes use of the system.custom target added via
 [this PR](https://github.com/AdoptOpenJDK/openjdk-tests/pull/2234))
 
+## Running The SSL Test Suites
```

Comment:
Picky comment, but given that this doesn't follow the normal pattern for test execution (yet) can we make this a twisty section for now so as not to take up too much extra space in the document, particularly with all the output that's being shown?
```suggestion
<details>
  <summary>Running the SSL test suites</summary>
```
(With a corresponding `</details`> at the end of the section)

This has made me think that it might be useful to have an example for the external tests which are run in containers too.

Reply:
Added a twisty section for the ssl-tests.

Reply:
Thank you!

Reply:
I'm working on the Dockerstatic Changes, so I'll add an additional twisty section to the FAQ, on running some of tests on a newly created container too.

Reply:
Is there a reason why we'd need additional documentation for that over and above what's already in the twisty? It shouldn't make a difference what environment you're running it should it?

Reply:
> Is there a reason why we'd need additional documentation for that over and above what's already in the twisty? It shouldn't make a difference what environment you're running it should it?

Having done some of this , no it doesn't :), I don think there's a need for any more doc updates., I'll resolve this once all the dockerstatic updates are done.

Reply:
Dockerstatic files updated , and tested.